app.js

```diff
@@ -78,6 +79,34 @@ app.post("/blog", (req, res) => {
       }
     },
   );
+
+  // Read the HTML file
+  fs.readFile('./frontend pages/blog-list-page.html', 'utf8', (err, html) => {
+    if (err) {
+      res.status(500).send('Error reading the HTML file');
+      return;
+    }
+
+    // Content to add
+    const newContent = `<h2> <a href='http://localhost:3000/blog-${newBlog.id}.html'>${newBlog.subject} </a></h2>`;
+
+    // Add the new content to the HTML
+    if (html.includes(newContent)) {
+
```

Comment:
Should this just return if true? or be changed to if false and get ride of the else?